src/app/components/editors/text-editor/text-editor.component.ts

```diff
@@ -15,19 +15,21 @@ export class TextEditorComponent implements OnInit {
     public description: string;
     constructor(
         private db: AngularFirestore,
-        private getUserIdService: GetUserIdService,
+        private getUserService: GetUserService,
     ) {}
     ngOnInit(): void {}
 
     public startUpload(textContent) {
         const { title, description } = textContent;
-        const userId = this.getUserIdService.getUserId();
+        const userId = this.getUserService.getUserId();
         this.db.collection('Posts').add({
```

Comment:
Could you please move the code which operates with db to the separated service? 
Like post service